pyEpiabm/pyEpiabm/core/microcell.py

```diff
@@ -123,27 +141,41 @@ def add_place(self, n: int, loc: typing.Tuple[float, float],
             self.cell.places.append(p)
             self.places.append(p)
 
-    def add_household(self, people: list):
+    def add_household(self, people: list, change_id: bool = True):
         """Adds a default :class:`Household` to Microcell and fills it with
         a number of :class:`Person` s.
 
         Parameters
         ----------
         people : list
             List of :class:`People` to add to household
+        change_id : bool
+            Boolean representing whether we wish to set the id of the people
+            of the household when the function is called or not
 
         """
         if len(people) != 0:
             household = Household(self, loc=self.location)
-            for person in people:
+            for i in range(len(people)):
+                person = people[i]
```

Comment:
```suggestion
            for i, person in enumerate(people):
```

This is a really handy construction!